rules/concepts.md

```diff
@@ -22,4 +23,59 @@ formerly implicit variables
 How to find and install.
 How to write will be on a later page.
 
-### Comprehensive Examples
\ No newline at end of file
+### Comprehensive Examples
+-->
+
+# Rules Concepts
+
+Thus far we've connected openHAB to devices through Things, modeled the devices with Items, discussed persistence and how to build the display to control your home.
+But all of that amounts to home control, not automation.
+To create home automation we need to define rules.
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+## What Are Rules
+
+You can think of rules as routines or behaviours for your smart home.
+
+Just like you stand up in the morning and have your breakfast, your smart home can have a routine to open the blinds when the sun rises.
+Another routine would be to turn all lights off when you leave home: openHAB can do that for you, no need to open your app and switch of the lights manually.
+You want to hear your favorite music when you arrive back at home? No problem, openHAB can do that for you.
+But rules can also remind you of things: you opened the window hours ago and forgot that is is open? openHAB can sent a notification to your phone.
```

Comment:
Put sentences on their own line. In both of these cases the "?" is the end of a sentence.